aep/0140.yaml

```
@@ -0,0 +1,17 @@
+rules:
+  aep-140-boolean-field-naming:
+    description: 'Boolean fields should omit the prefix "is".'
```

I think we should use OAS terminology over protobuf terminology wherever these differ
```suggestion
  aep-140-boolean-property-naming:
    description: 'Boolean properties should omit the prefix "is".'
```